---
layout: archive
---

{{ content }}


<h3 class="archive__subtitle">{{ site.data.ui-text[site.locale].recent_posts | default: "ABOUT" }}</h3>


I am a highly collaborative computational biologist passionate about tackling
difficult biological analyses with a focus on genomics, multi-omics,
statistics, machine learning, and software development. I emphasize
revealing biological insight by conceptually understanding and modeling systems
with multi-omics data, though I am also excited to tackle recalcitrant problems
with machine learning. My methodological background is centered on
big NGS data analysis (genomics/transcriptomics), mass spectrometry, phylogenomics,
and discovering/optimizing genetic pathways underlying phenotypes-of-interest.
My expertise has extensively been applied to microbial natural product/biological drug
discovery, and I have directly worked with eukaryotes and prokaryotes. I am
passionate about presenting and training, which includes invited teaching and
<a href="https://youtu.be/5FVbRDy-nBU?si=NUouztIDBSnR9nf8&t=3143s">workshop
presentations</a> for technical expertise and general
botany/mycology education.

<br><br>

My career includes both industry and academic experience, with 5 years in
natural product drug discovery analytical chemistry, followed by 6 years in
computational genomics (doctoral, post-doctoral, industry consultation). My
PhD in <a href="http://u.osu.edu/slot.1">Jason Slot's</a> lab culminated in 
1) an evolution-informed natural product biosynthesis targeting platform that <a
href="https://doi.org/10.1093/nar/gkae625">can recover
biosynthetic pathways with increased accuracy and recovery compared to published 
standards in the field</a>; and 2) a platform for <a
href="https://www.biorxiv.org/content/10.1101/2023.09.08.556886v1">automating
large-scale comparative phylogenomics</a> that serves as the genomics
database for Ohio Supercomputer Center and University of Wisconsin-Madison high
performance computing cluster. I am driven to address
applied questions in industry, where I have unveiled cryptic biosynthetic
pathways and <a href="https://www.pnas.org/doi/10.1073/pnas.2214076120">identified
elusive flavor genes in shiitake mushrooms</a> by integrating transcriptomic 
and phylogenomic data. I employed another tailored genome-guided strategy to <a
href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0289280">pinpoint
clusters of colocalized genes that produce analogs of the psychoactive drug,
LSD</a>.

<br><br>

<img src="https://github.com/xonq/xonq.github.io/blob/master/assets/images/mycomics.png?raw=true" title="mycomics, llc: consulting and contracting (yes, that's a phylogeny of Fungi)">


{% if paginator %}
  {% assign posts = paginator.posts %}
{% else %}
  {% assign posts = site.posts %}
{% endif %}

{% assign entries_layout = page.entries_layout | default: 'list' %}
<div class="entries-{{ entries_layout }}">
  {% for post in posts %}
    {% include archive-single.html type=entries_layout %}
  {% endfor %}
</div>

{% include paginator.html %}
